cmd/root.go

```diff
@@ -114,12 +111,8 @@ func init() {
 
 	rootCmd.SetVersionTemplate("{{with .Name}}{{printf \"%s \" .}}{{end}}{{printf \"v%s\" .Version}}\n")
 
-	l := logger.GetLogger()
-
 	// initialize all subcommands
 	rootCmd.AddCommand(observe.New())
-	rootCmd.AddCommand(serve.New(l))
```

Comment:
This PR completely removes our standalone server required to run Hubble with Cilium 1.6 and 1.7. It this intended?

Reply:
Yes, we have the 0.5.0 release out there to support the standalone mode.

Reply:
I've added a commit to ping images to the 0.5.0 release

Reply:
Does this mean we have to branch out v0.5 for bugfixes, etc? It seems more work than just keep the `serve` command in the tree for now (maybe with a deprecation notice and/or marking it hidden).

Reply:
Ah okay, yeah I was also under the impression we would keep support in the `master` tree, but having a `v0.5` branch works for me as well. 

It will signal very clearly that we are not supporting standalone mode anymore besides urgent fixes. But yes, it does mean that backports to v0.5 will involve a lot of manual labor.

(sidenote: if we want to keep support in master, PR #218 allows us to enable/disable the `serve` command at build-time)

Reply:
Yes, we will have to branch out if we want to push bugfixes into 0.5.0. We do the same with Cilium.

If we keep the code around, somebody will make changes.

Reply:
Would it be possible then to push this serve command removal to a separate commit so that we can branch out from the server-side code removal commit in order to pick up fixes that have already been committed to master since v0.5 was tagged?

Reply:
Are you asking to create a commit which changes the Hubble to use the server-side code in cilium/cilium first before removing all of that code?

Reply:
Yes. I think it would make it easier to pickup fixes for hubble server that are made in cilium's repository. We'll have to backport them manually otherwise.

Reply:
Are you OK to do this work once we need to push a fix to the 0.5.0 branch? We can do such a change as the first commit if we ever have to branch off. It's also a bit unclear how that would look like from a versioning perspective. The server side code in cilium is only in master, unreleased, would we have the 0.5 Hubble branch point to an arbitrary Cilium commit in unreleased master?

Reply:
> Are you OK to do this work once we need to push a fix to the 0.5.0 branch? 

Yes. I just want to make sure that we keep the overhead of maintaining the 0.5 branch at a minimum.